master/static/migrations/20231208132223_drop-raw-checkpoints.tx.up.sql

```diff
@@ -0,0 +1,16 @@
+DROP TABLE raw_checkpoints CASCADE;
```

Comment:
nit: can we avoid using ```CASCADE``` just so we are explicit about what we need to drop before being able to drop ```raw_checkpoints```